extension/fts/src/utils/fts_utils.cpp

```diff
@@ -63,6 +72,10 @@ std::vector<std::string> FTSUtils::stemTerms(std::vector<std::string> terms,
         if (isConjunctive && checker.isStopWord(term)) {
             continue;
         }
+        if (hasWildcardPattern(term)) {
```

Comment:
i think this can be a static information:
If neither of the keyword has wildcard pattern in it, we can avoid doing a gds scan to find the terms since it can be expensive.
Otherwise, we can do a full table scan and match the wildcard pattern.